websites/D/Desu-Online/presence.ts

```diff
@@ -0,0 +1,166 @@
+const presence = new Presence({
+		clientId: "580032576434077707",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/KEkxfKO.png",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, search, host } = document.location;
+	if (host === "desu-online.pl") {
```

Comment:
Based on `metadata.json`, since this presence only runs on only one hostname, this if statement is not needed.

Reply:
You mean premid takes the `url` from metadata.json and executes the code based on it, so `if (host === "desu-online.pl")` is redundant since site only has one address?